tools/accuracy_checker/accuracy_checker/postprocessor/crop_segmentation_mask.py

```diff
@@ -54,10 +54,22 @@ def configure(self):
 
     def process_image(self, annotation, prediction):
         for target in annotation:
-            target.mask = Crop3D.crop_center(target.mask, self.dst_height, self.dst_width, self.dst_volume)
+            shape = len(target.mask.shape)
+            if shape == 3:
+                target.mask = Crop3D.crop_center(target.mask, self.dst_height, self.dst_width, self.dst_volume)
+            if shape == 2:
+                target.mask = Crop.process_data(target.mask, self.dst_height, self.dst_width, None, False, True, {})
+            else:
+                raise ValueError("'arr' does not have a suitable array shape for any mode.")
 
         for target in prediction:
-            target.mask = Crop3D.crop_center(target.mask, self.dst_height, self.dst_width, self.dst_volume)
+            shape = len(target.mask.shape)
+            if shape == 3:
```

Comment:
hmm... what if prediction will be represented before argmax (H, W, C, where C is number of channels)?